src/components/contextual/stake/StakePreview.vue

```diff
@@ -105,125 +135,152 @@ const poolShareData = computed(() => {
  */
 onBeforeMount(async () => {
   await loadApprovalsForGauge();
+
+  stakedShares.value = await getStakedShares();
```

Comment:
Receiving `stakedShares` from contract pending response from Fabio regarding whether `balance` from gauge subgraph is cumulative or not.